assets/js/components/DetailsPermaLinks.js

```diff
@@ -49,6 +49,7 @@ export default function DetailsPermaLinks( { title, path, serviceURL } ) {
 	return (
 		<Fragment>
 			<Link
+				className="googlesitekit-font-weight-medium"
```

Comment:
I've gone ahead and set the medium font weight here so it's applied to the desktop viewport and WP dashboard "top content" widget as well, as per this [Slack thread](https://fueled10up.slack.com/archives/C01CA828ZPZ/p1732616012510899), as I think it's most likely that we'll go ahead with this approach.

If it turns out we decide not to, it will be quick to amend in a followup issue.